scripts/floods/raster_to_csv.py

```diff
@@ -0,0 +1,1149 @@
+    'ignore_csv': _FLAGS.ignore_csv,
+    'allow_csv': _FLAGS.allow_csv,
+    'limit_points': _FLAGS.limit_points,
+
+    # Output settings
+    'output_csv': _FLAGS.output_csv,
+    'output_mode': _FLAGS.output_mode,
+    'output_columns': _FLAGS.output_columns,
+    'output_date': _FLAGS.output_date,
+    'output_precision': _FLAGS.output_precision,
+    'output_s2_place': _FLAGS.output_s2_place,
+    'contained_in_s2_level': _FLAGS.contained_in_s2_level,
+
+    # Processing parameters
+    's2_level': _FLAGS.s2_level,
+    # default aggregation for data values mapped to the same s2-cell+date
+    # that can be one of: min, max, mean, sum.
+    # For columns specific aggregation methods, use config:'input_data_filter'.
+    'aggregate': _FLAGS.aggregate,
+    # generate data for all s2 cells from --s2_level upto this level.
+    'aggregate_s2_level': _FLAGS.aggregate_s2_level,
+    # Default cell area if constant for the whole data set.
+    'default_cell_area': _FLAGS.default_cell_area,
+    # Default point/cell width/height for ~1sqkm at equator, lower near poles.
+    'default_cell_width': 0.009,
+    'default_cell_height': 0.009,
+    # rename columns <input-name>: <output-name>
+    'rename_columns': {
+        'acq_date': 'date',
+        'band:0': 'water',
+    },
+    # Unused
+    'parallelism': _FLAGS.parallelism,
+
+    # filter settings per column for data
+    # input (pre-aggregation) and output (post-aggregation) of the form:
+    # {
+    #   '<column>' : {
+    #     'min': <NN.NNN>, # Minimum value for <column>
+    #     'max': <NN.NNN>, # Maximum value for <column>
+    #     'aggregate': 'sum', # one of 'min','max','sum','mean'
+    #   },
+    #   ...
+    # }
+    'input_data_filter': {
+        # Add up area for points added to an s2cell.
+        'area': {
+            'aggregate': 'sum',
+        },
+        # 'water': {  # band:0
+        #     'min': 1.0
+        #},
+    },
+    'output_data_filter': {
+        #   'area': {
+        #       'min': 1.0,  # Minimum area in sqkm after aggregation
+        #   },
+    },
+
+    # Debug settings
+    'debug': _FLAGS.debug,  # Enable debug logs
+    'log_every_n':
+        _FLAGS.log_every_n,  # Show counters when counter increments by N
+    'log_every': 'input_points',  # Counter to check for log_every_n
+}
+
+_DEFAULT_COLUMNS = set({'latitude', 'longitude', 's2CellId', 'date'})
+
+
+def load_config(config: str, default_config: dict = _DEFAULT_CONFIG) -> dict:
+    '''Load config from string or file.'''
+    config_dict = dict(default_config)
+    config_str = config
+    if isinstance(config, str):
+        if os.path.exists(config):
+            with open(config, 'r') as config_file:
+                config_str = config_file.read()
+        if config_str:
+            config_dict.update(ast.literal_eval(config_str))
+    elif isinstance(config, dict):
+        config_dict.update(config)
+    logging.info(f'Using config: {config_dict}')
+    return config_dict
+
```

Comment:
there is a similar function in the other py file? can these be shared in a util?

generally wonder if we should break up some of the code here into helpers?  a 1000+ line py file is big :)